frontend/projects/upgrade/src/app/core/analysis/store/analysis.actions.ts

```diff
@@ -33,7 +34,10 @@ export const actionUpsertMetricsSuccess = createAction(
 
 export const actionUpsertMetricsFailure = createAction('[Analysis] Upsert Metrics Failure');
 
-export const actionExecuteQuery = createAction('[Analysis] Execute Query', props<{ queryIds: string[] }>());
+export const actionExecuteQuery = createAction(
```

Comment:
You don't need to send the state information from the frontend. You can get that from the queryId itself.

Reply:
code updated